ee/localserver/krypto-ec-middleware.go

```diff
@@ -90,6 +92,15 @@ type kryptoEcMiddleware struct {
 
 func newKryptoEcMiddleware(slogger *slog.Logger, localDbSigner crypto.Signer, counterParty ecdsa.PublicKey, presenceDetector presenceDetector, tenantMunemo string) *kryptoEcMiddleware {
 	atomicMunemo := atomic.NewString(tenantMunemo)
+
+	// Set up our callback queue with a worker to send callbacks. The callback queue has a buffer
+	// because we never want to block on sending to the queue.
+	callbackQueue := make(chan *http.Request, 100)
```

Comment:
nit
```suggestion
	callbackQueue := make(chan *http.Request, maxCallbackQueueSize)
```

Reply:
I figured we'd still want the way bigger queue capacity just to be very cautious. Maybe for clarity, I could do `maxDesiredCallbackQueueSize = 10` and `callbackQueueCapacity = 100` consts?

Reply:
sure that works for me!